source/arriving/setup_computer.rst

```diff
@@ -152,12 +152,52 @@ In any case, please refer to the `Github page <https://github.com/scilus/scilpy>
 Containers
 """"""""""
 
-TODO
+Container technologies allow applications and services to be packaged into isolated, self-contained units that can be deployed on any cloud or local host. They are like tiny isolated computers running on other computers. For more information, visit the guide on containers and virtual machines (:ref:`ref_virtualmachines`). A container bundling all the dependencies required for running any flow (see section :ref:`ref_flow`) developped in the lab is available `here <https://hub.docker.com/r/scilus/scilus>`__. In the lab, we use Docker and Apptainer (formerly Singularity) containers, which we describe below.
+
+Docker
+------
+The most popular container technology is undoubtedly Docker. Docker is supported on Mac, Windows and Linux, although it'd seem like Docker running a Linux instance do not work well on MacOS with M1/M2 CPU. To install Docker, follow this documentation: `<https://docs.docker.com/engine/install/ubuntu>`__.
```

Comment:
You prefer fully visible links rather than a clickable "documentation"?

Reply:
No I thought it was what you preferred. Will fix.

Reply:
I just didn't like the ": here"!